utils/toast.js

```diff
@@ -0,0 +1,20 @@
+import Swal from 'sweetalert2'
+
+export default function toast(type, message) {
+  const Toast = Swal.mixin({
+    toast: true,
+    position: 'top',
+    showConfirmButton: false,
+    timer: 3000,
+    timerProgressBar: true,
+    didOpen: (toast) => {
+      toast.addEventListener('mouseenter', Swal.stopTimer)
+      toast.addEventListener('mouseleave', Swal.resumeTimer)
+    },
+  })
```

Comment:
we could add something like `showCloseButton` so that the toaset can be closed if the user don't want to wait till auto close.

Reply:
Thanks Bhai, Nice catch, resolved.